src/styles/styles.js

```diff
@@ -2917,6 +2917,43 @@ const styles = {
         fontSize: variables.fontSizeXXLarge,
         letterSpacing: 4,
     },
+
+    footer: {
+        backgroundColor: themeColors.midtone,
+    },
+
+    footerWrapper: {
+        fontSize: variables.fontSizeNormal,
+        paddingTop: 64,
+        paddingHorizontal: 32,
+        maxWidth: 1100,
```

Comment:
Could you please add a comment for why 1100 is the specific value? Does it relate to an existing breakpoint?